src/hardware/mixer.cpp

```diff
@@ -2322,6 +2340,18 @@ void MIXER_Init(Section *sec)
 	configure_reverb(section->Get_string("reverb"));
 	configure_chorus(section->Get_string("chorus"));
 
+	// Initialise compressor
+	const auto threshold_db    = -3.0f;
+	const auto ratio           = 4.0f;
+	const auto release_time_ms = 5000.0f;
+	const auto rms_window_ms   = 10.0;
+
+	mixer.compressor.Configure(mixer.sample_rate,
+	                           threshold_db,
+	                           ratio,
+	                           release_time_ms,
+	                           rms_window_ms);
```

Comment:
In the compressor.cpp, if it's possible to write up a bit how it works - and especially how these config values influence the compressors behavior, it would help users/maintainers.

Reply:
Yeah sure, I guess I'm just assuming some basic familiarity with compressors from the reader, but probably I shouldn't 😁 (I've just been dealing with audio for too long to realise, my bad)

Reply:
Added a brief description and a link to a Wikipedia page on dynamic-range compression that describes the standard controls in great detail.